reana_client/cli/workflow.py

```diff
@@ -131,9 +131,11 @@ def workflow_workflows(ctx, sessions, _filter, output_format, access_token,
             workflow['name'] = name
             workflow['run_number'] = run_number
             if type == 'interactive':
-                workflow['session_uri'] = '{reana_server_url}{path}'.format(
-                    reana_server_url=ctx.obj.reana_server_url,
-                    path=workflow['session_uri'])
+                workflow['session_uri'] = \
```

Comment:
👍 , what do you think about extracting this formatting logic to a function? We have it also in the [`open` command](https://github.com/reanahub/reana-client/blob/master/reana_client/cli/workflow.py#L877) so we might risk having diverging formats if we forget.